bin/ch/Helpers.cpp

```diff
@@ -665,7 +655,7 @@ void CALLBACK Helpers::TTInitializeForWriteLogStreamCallback(size_t uriByteLengt
     Helpers::CleanDirectory(uriByteLength, uriBytes);
 }
 
-JsTTDStreamHandle CALLBACK Helpers::TTCreateStreamCallback(size_t uriByteLength, const byte* uriBytes, const char* asciiResourceName, bool read, bool write)
+JsTTDStreamHandle CALLBACK Helpers::TTCreateStreamCallback(size_t uriByteLength, const byte* uriBytes, const char* asciiResourceName, bool read, bool write, byte** relocatedUri, size_t* relocatedUriLength)
```

Comment:
Are these parameters supposed to be used? Should they be at least default-initialized?